backend/src/agents/materiality_agent.py

```diff
@@ -0,0 +1,36 @@
+import json
+from pathlib import Path
+import logging
+
+from src.llm import LLMFile
+from src.agents import Agent
+from src.prompts import PromptEngine
+
+engine = PromptEngine()
+logger = logging.getLogger(__name__)
+
+
+class MaterialityAgent(Agent):
+    async def list_material_topics(self, company_name: str) -> dict[str, str]:
+        with open('./library/catalogue.json') as file:
+            catalogue = json.load(file)
+            files_json = await self.llm.chat(
```

Comment:
Nitpick: Worth unindenting from here to end of method, so we do the file read, close it at end of `with` and then do the llm calls etc?